docs/about.md

```diff
@@ -8,7 +16,7 @@ We want `plone.api` to be developed with [PEP 20](https://peps.python.org/pep-00
 >
 > Readability counts.
 >
-> There should be one-- and preferably only one --obvious way to do it.
+> There should be one – and preferably only one – obvious way to do it.
```

Comment:
These should be "em dashes", not hyphens. Also most style guides prefer no spaces around em dashes. The Zen of Python uses double-dashes instead of em dashes, but modern style guides frown upon that unless your keyboard does not easily support em dashes.

```suggestion
> There should be one—and preferably only one—obvious way to do it.
```